packages/login/network/BitcoinMiddlewareNode.js

```diff
@@ -78,6 +86,11 @@ export default class BitcoinMiddlewareNode extends AbstractNode {
     }
   }
 
+  async getFeeRate () {
```

Comment:
remove `async`

Reply:
See comment in the next row
Also see implementation of the same method in BitcoinBlockexplorerNode.
It is important to know that this implementation can be async.

Won't fix